contrib/testnet/monitoring/dashboards/details-netdb.json

```diff
@@ -0,0 +1,730 @@
+                            "text": "kovri_testnet_016",
+                            "value": "kovri_testnet_016"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_017",
+                            "value": "kovri_testnet_017"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_018",
+                            "value": "kovri_testnet_018"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_019",
+                            "value": "kovri_testnet_019"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_020",
+                            "value": "kovri_testnet_020"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_021",
+                            "value": "kovri_testnet_021"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_022",
+                            "value": "kovri_testnet_022"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_023",
+                            "value": "kovri_testnet_023"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_024",
+                            "value": "kovri_testnet_024"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_025",
+                            "value": "kovri_testnet_025"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_026",
+                            "value": "kovri_testnet_026"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_027",
+                            "value": "kovri_testnet_027"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_028",
+                            "value": "kovri_testnet_028"
+                        },
+                        {
+                            "selected": false,
+                            "text": "kovri_testnet_029",
+                            "value": "kovri_testnet_029"
```

Comment:
>kovri_testnet_029

Is there any way we can generate these entries dynamically? For the cases where instance number 029 doesn't exist? What will happen if that instance doesn't exist? What happens if there is 030?

Reply:
> Is there any way we can generate these entries dynamically?

Yes. Basically, they are generated by a query :

    SHOW TAG VALUES WITH KEY = \"instance\"

So it should adapt to any number of instances, but i did indeed forgot to activate the auto refresh for `Instance Overview` (fixed now).

For the other dashboards, i disabled the auto refresh to keep a set of predefined filters (otherwise it auto selects the first element in the result, bypassing the predefined default filter and these views are more suited to compare multiple instances).